apps/expert/lib/expert/engine_node.ex

```diff
@@ -221,7 +223,7 @@ defmodule Expert.EngineNode do
 
       case Expert.Port.elixir_executable(project) do
         {:ok, elixir, env} ->
-          GenLSP.info(lsp, "Found elixir for #{project_name} at #{elixir}")
+          Expert.log_info(lsp, "Finding or building engine at #{elixir}")
```

Comment:
`elixir` here is the path of the executable. So maybe something like `"Found elixir executable at #{elixir}"`?